docs/product/alerts/uptime-monitoring/index.mdx

```diff
@@ -4,31 +4,36 @@ sidebar_order: 50
+
+## Set Up
+
+Uptime is [automatically configured](/product/alerts/uptime-monitoring/automatic-detection/) as a new alert for the most most frequently encountered hostname in all URLs of your error data, ensuring continuous monitoring of your most critical hostname right out of the box.
+
+You can also [create uptime monitoring alerts](/product/alerts/create-alerts/) for specific URLs, fully customizable with request details such as the HTTP method, headers, and body.
 
 ## Uptime Check Criteria
 
-Our uptime monitoring system verifies the availability of your URLs
-by performing GET requests at regular 5-minute intervals.
-For a URL to be considered up and running, the response must meet the following criteria:
+Our uptime monitoring system verifies the availability of your URLs by performing HTTP requests at regular, pre-configured intervals. For a URL to be considered up and running, the response must meet the following criteria:
 
 1. **Successful Response (2xx Status Codes):**
-The URL must return an HTTP status code in the 200–299 range, indicating a successful request.
+   The URL must return an HTTP status code in the 200–299 range, indicating a successful request.
 2. **Automatic Handling of Redirects (3xx Status Codes):** Sentry will follow redirects for URLs returning an HTTP status code in the 300–399 range and verify that the final destination URL returns a successful response. This ensures that redirects won't falsely trigger downtime alerts.
 3. **Timeout Setting:** Each request has a timeout threshold of 10 seconds.
-If the server doesn't respond within this period, the check will be marked as a failure,
-indicating a potential downtime or performance issues.
+   If the server doesn't respond within this period, the check will be marked as a failure,
+   indicating a potential downtime or performance issues.
 4. **DNS Issue Detection:** Our monitoring also includes the detection of DNS resolution issues.
-If a DNS issue is detected, the check will be marked as a failure,
-allowing you to address the underlying connectivity problems.
+   If a DNS issue is detected, the check will be marked as a failure,
+   allowing you to address the underlying connectivity problems.
 
 ## Notifications
 
 An uptime alert continuously monitors the configured URL with the criteria defined above. If a failure is detected,
-a new [uptime issue](/product/issues/issue-details/uptime-issues/) with failed check and related errors details will be created.
+a new [uptime issue](/product/issues/issue-details/uptime-issues/) with the failed check and related error details will be created. To avoid triggering false alerts due to transient issues like network glitches, a new issue is created only after detecting a minimum of three consecutive failures following the initial downtime detection.
```

Comment:
```suggestion
a new [uptime issue](/product/issues/issue-details/uptime-issues/) with the failed check and related error details will be created. To avoid triggering false alerts due to transient issues like network glitches, new issues will only be created after a minimum of three consecutive failures have been detected, following the initial downtime detection.
```